tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -61,12 +66,40 @@
 # TODO: Update the table after fix. Target is < 5ms.
 LOAD_LATENCY_BASELINES = {
     "x86_64": {
-        "2vcpu_256mb.json": {"target": 9},
-        "2vcpu_512mb.json": {"target": 9},
+        "4.14": {
+            "sync": {
+                "2vcpu_256mb.json": {"target": 9},
+                "2vcpu_512mb.json": {"target": 9},
+            }
+        },
+        "5.10": {
+            "sync": {
+                "2vcpu_256mb.json": {"target": 60},
+                "2vcpu_512mb.json": {"target": 60},
+            },
+            "async": {
+                "2vcpu_256mb.json": {"target": 190},
+                "2vcpu_512mb.json": {"target": 190},
+            },
+        },
     },
     "aarch64": {
-        "2vcpu_256mb.json": {"target": 3},
-        "2vcpu_512mb.json": {"target": 3},
+        "4.14": {
+            "sync": {
+                "2vcpu_256mb.json": {"target": 2},
```

Comment:
This seems like a marked improvement?

Reply:
Yup, it seems so. I checked git history and the first values gathered when we added support for ARM were 3 ms.

Reply:
This is good news, but I just want to make a side comment on the granularity. If the values are so low, it is hard to say if this was a 33% improvement or if rounding is overstating the impact. I.e. Was this before 2.51 and it rounded up to 3, and now it's 2.49 and rounds down to 2? Could we use floats or some higher resolution timer?